src/exampleTypes/ExpandableTagsQuery/index.js

```diff
@@ -11,7 +11,14 @@ let collapsedStyle = {
   overflowY: 'auto',
 }
 
-let ExpandableTagsQuery = ({ measureRef, contentRect, collapse, ...props }) => (
+let ExpandableTagsQuery = ({
+  // eslint-disable-next-line no-unused-vars
+  measure,
```

Comment:
Why destructure measure if it's not used? Is it so it's omitted on `props` before passing to `TagsQuery`?

Reply:
Yeah, otherwise it gets passed to an `input` inside of `TagsQuery` and shows React error

Reply:
I've used `_.omit` instead